docs/pt/docs/fastapi-people.md

```diff
@@ -0,0 +1,138 @@
+
+Eles contribuíram com o código-fonte, documentação, traduções, etc. 📦
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_contributors %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Pull Requests: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+Existem muitos outros contribuidores (mais de uma centena), você pode vê-los todos na 
+
+Existem muitos outros contribuidores (mais de cem), você pode vê-los todos na <a href="https://github.com/tiangolo/fastapi/graphs/contributors" class="external-link" target="_blank">página FastAPI GitHub Contributors</a>. 👷
+
+## Principais Revisores
+
+Esses usuários são os **Principais Revisores**. 🕵️
+
+### Críticas para traduções
+
+Eu só falo algumas línguas (e não muito bem 😅). Portanto, os revisores são os que têm o [**poder de aprovar traduções**](contributing.md#translations){.internal-link target=_blank} da documentação. Sem eles, não haveria documentação em vários outros idiomas.
+
+---
+
+Os **Principais Revisores** revisaram a maioria dos Pull Requests de outros, garantindo a qualidade do código, da documentação e, principalmente, das **traduções**.
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.top_reviewers %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Revisões: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Patrocinadores
+
+Estes são os **patrocinadores**. 😎
+
+Eles estão apoiando meu trabalho com **FastAPI** (e outros) por meio do <a href="https://github.com/sponsors/tiangolo" class="external-link" target="_blank">GitHub Sponsors</a>.
+
+{% if people %}
+<div class="user-list user-list-center">
+{% for user in people.sponsors %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Sobre os dados - detalhes técnicos
+
+A intenção desta página é destacar o esforço da comunidade para ajudar os outros.
+
+Especialmente incluindo esforços que normalmente são menos visíveis e, em muitos casos, mais árduos, como ajudar outras pessoas com problemas e revisar Pull Requests com traduções.
+
+Os dados são calculados a cada mês, você pode ler o <a href="https://github.com/tiangolo/fastapi/blob/master/.github/actions/people/app/main.py" class="external-link" target="_blank">código-fonte aqui</a>.
+
+Eu também me reservo o direito de atualizar o algoritmo, seções, limites, etc (apenas no caso 🤷).
```

Comment:
`apenas no caso` didn't make sense to me, I believe that the following suggestion makes it better.  

```suggestion
Eu também me reservo o direito de atualizar o algoritmo, seções, limites, etc (só por garantia 🤷).
```